tools/run_tests/artifact_targets.py

```diff
@@ -99,8 +103,47 @@ def build_jobspec(self):
     if self.platform == 'linux':
       if self.arch == 'x86':
         environ['SETARCH_CMD'] = 'linux32'
+      # Inside the manylinux container, the python installations are located in
+      # special places...
+      environ['PYTHON'] = '/opt/python/{}/bin/python'.format(self.manylinux_build)
+      environ['PIP'] = '/opt/python/{}/bin/pip'.format(self.manylinux_build)
+      # Our docker image has all the prerequisites pip-installed already.
+      environ['SKIP_PIP_INSTALL'] = '1'
+      # Platform autodetection for the manylinux1 image breaks so we set the
+      # defines ourselves.
+      # TODO(atash) get better platform-detection support in core so we don't
+      # need to do this manually...
+      environ['CFLAGS'] = " ".join([
```

Comment:
The `manylinux1` spec is really just an old `__linux__` with a few requirements on `__GLIBC_*_` and `__GNUC_*_` ([and a few other things that I don't think we care about](https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy)). Or should we just punt on being graceful about it for now?